layouts/partials/search-results.html

```diff
@@ -1,5 +1,23 @@
+<div class="container">
+  <h1 id="search-title">Search</h1>
+  <ul id="results"></ul>
+</div>
+<script>
+  const params = new URLSearchParams(window.location.search);
+  const queryParam = params.get('query');
+  console.log("Query param is " + params.get('query'));
+  if(queryParam !== null && queryParam.length > 0)
+  {
+    $(".search-input-view").show();
+    $('#search-title').text('Searching for ' + '"' + queryParam + '" ...');
+    $("#search-input").val(queryParam);
+  }
+  // close off this script block so it gets evaluated before the heavy lifting
+</script>
 <script>
-  window.store = {
+  if(queryParam !== null && queryParam.length > 0)
+  {
+    window.store = {
```

Comment:
We should make a future task to avoid building the index every time and do something more like this:  https://palant.info/2020/06/04/the-easier-way-to-use-lunr-search-with-hugo/